Dockerfile-multistage

```diff
@@ -43,17 +43,17 @@ RUN curl -fsSL https://deb.nodesource.com/setup_$NODE_VERSION | bash - && \
     underscore@1.12.0 \
     underscore-deep-extend@1.1.5
 
-ENV NODE_PATH /usr/lib/node_modules
+ENV NODE_PATH=/usr/lib/node_modules
 
 # Add runner user so we don't run as root
 RUN	useradd -u 1000 -ms /bin/bash runner && \
   chown -R runner:runner /usr/local/tomcat/
 
 # Allow certificates to be added by runner
-RUN chgrp runner /usr/local/openjdk-11/lib/security/cacerts && \
-    chmod g+w /usr/local/openjdk-11/lib/security/cacerts && \
+RUN chgrp runner $JAVA_HOME/lib/security/cacerts && \
+    chmod g+w $JAVA_HOME/lib/security/cacerts && \
     mkdir -p /usr/lib/jvm/java/jre/lib/security && \
-    ln -s /usr/local/openjdk-11/lib/security/cacerts /usr/lib/jvm/java/jre/lib/security/cacerts
+    ln -s $JAVA_HOME/lib/security/cacerts /usr/lib/jvm/java/jre/lib/security/cacerts
```

Comment:
This has to do with where some docker compose files are importing certs into the /usr/lib/jvm/java/jre/lib/security/cacerts locations so leaving that part the same